setup.sh

```diff
@@ -195,3 +187,25 @@ if [ -n "${SETUP_LXD-}" ]; then
         fi
     fi
 fi
+
+always_activate() {
+  if [ ! -f ~/.bashrc ] || ! grep 'source.*activate' ~/.bashrc; then
+    echo '# Activate ansible virtualenv' >> ~/.bashrc
+    echo "source $ANSIBLE_SETUP_DIR/bin/activate" >> ~/.bashrc
+    echo 'export ANSIBLE_STDOUT_CALLBACK=debug' >> ~/.bashrc
+    echo '!! You need to login again for changes to take effect !!'
+  fi
+}
+
+if [[ $- == *i* ]]; then
+  while true; do
+    read -p "Do you wish to activate ansible setup in your shell ? "  yn
+    case $yn in
+      y ) always_activate; break;;
+      n ) exit;;
+      * ) echo "Please answer y or n.";;
+    esac
+  done
+elif [ "${ALWAYS_ACTIVATE-1}" = "1" ]; then
+    always_activate
+fi
```

Comment:
OK alors si tu veux merger vas y essaye juste de rester dispo si y'a besoin
de faire le pompier à cause du patch ;)

On Sep 7, 2016 9:05 PM, "Toopy" <notifications@github.com> wrote:

> In setup.sh
> <https://github.com/novafloss/ansible-setup/pull/4#discussion_r77884273>:
>
> > +    echo '!! You need to login again for changes to take effect !!'
> > +  fi
> > +}
> > +
> > +if [[ $- == *i* ]]; then
> > +  while true; do
> > +    read -p "Do you wish to activate ansible setup in your shell ? "  yn
> > +    case $yn in
> > +      y ) always_activate; break;;
> > +      n ) exit;;
> > +      * ) echo "Please answer y or n.";;
> > +    esac
> > +  done
> > +elif [ "${ALWAYS_ACTIVATE-1}" = "1" ]; then
> > +    always_activate
> > +fi
>
> ALWAYS_ACTIVATE est une nouvelle variable et sa valeur par défaut est 1,
> du coup, sauf erreur, le comportement précédent est conservé nan ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/novafloss/ansible-setup/pull/4/files/97f9cc4f320ecea237f5e3df967622bfdf2fd680#r77884273>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFxrPdnkFpreQQYUgQBse2vRpydd8thks5qnwrdgaJpZM4JxnF7>
> .
>